l1-contracts/test/foundry/unit/concrete/Bridgehub/experimental_bridge.t.sol

```diff
@@ -641,8 +879,15 @@ contract ExperimentalBridgeTest is Test {
         mockChainContract.setBridgeHubAddress(address(bridgeHub));
         assertTrue(mockChainContract.getBridgeHubAddress() == address(bridgeHub));
 
-        vm.txGasPrice(0.05 ether);
+        if (msgValue != mockMintValue) {
+            vm.deal(randomCaller, msgValue);
+            vm.expectRevert("Bridgehub: msg.value mismatch 1");
+            vm.prank(randomCaller);
+            bridgeHub.requestL2TransactionDirect{value: msgValue}(l2TxnReqDirect);
+        }
 
+        vm.deal(randomCaller, l2TxnReqDirect.mintValue);
+        vm.txGasPrice(0.05 ether);
```

Comment:
Good approximation, but ideally we would this value to be fuzzed as well. You can upper bound the values to be in a realistic range.

Reply:
Gas is now fuzzed and bounded